lms/tasks/email_digests.py

```diff
@@ -134,23 +121,6 @@ def send_instructor_email_digests(
         with request.tm:
             digest_service = request.find_service(DigestService)
 
-            try:
-                digest_service.send_instructor_email_digests(
-                    h_userids, updated_after, updated_before, **kwargs
-                )
-            except SendDigestsError as err:
-                self.retry(
-                    kwargs={
-                        **self.request.kwargs,
-                        "h_userids": list(err.errors.keys()),
-                    },
-                    countdown=_retry_countdown(self.request),
-                )
-            except Exception as exc:  # pylint:disable=broad-exception-caught
-                LOG.exception(exc)
-                report_exception(exc)
-                self.retry(countdown=_retry_countdown(self.request))
-
-
-def _retry_countdown(celery_request):
-    return (3600 * (celery_request.retries + 1)) + random.randint(0, 900)
+            digest_service.send_instructor_email_digests(
+                h_userids, updated_after, updated_before, **kwargs
+            )
```

Comment:
The task just doesn't need to do any exception handling around this method call: any exceptions will just be raised and Celery will re-try the task after a delay. This will result in some unnecessary processing because `send_instructor_email_digests()` will re-generate the data for any emails that it did already manage to successfully send but we shouldn't actually send any duplicates because of the `task_done` table